public/plugin.ts

```diff
@@ -46,6 +48,7 @@ export interface AnomalyDetectionSetupDeps {
 
 export interface AnomalyDetectionStartDeps {
```

Comment:
I think we might have to move all these to the setup deps because they are being used from the flyout which is rendered on top of OSD. Since the ad-dashboard plugin hasn't been started these don't get called if they are in the start. According change needs to be made in line 92 `start` method

Reply:
I can do this in a follow up PR to reduce scope of change here